CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 ### Chore & Maintenance
 
+- `[*]` Add documentation and tests related to auto-mocking ([#8086](https://github.com/facebook/jest/pull/8099))
```

Comment:
Can you move this up under master?

(Might have to merge in master or rebase first)

Reply:
Synced with upstream master and moved the changelog line under the master heading. Thanks for the feedback and help with this @SimenB !